jobs/async-upload/job/config.py

```diff
@@ -18,66 +19,72 @@ def _parser() -> cap.ArgumentParser:
     # --- source ---
     # s3
     # TODO: We should be able to infer the type from the credentials provided, therefore no default needed
-    p.add("--source-type", choices=["s3", "oci"], default="s3")
-    p.add("--source-aws-bucket")
-    p.add("--source-aws-key")
-    p.add("--source-aws-region")
-    p.add("--source-aws-access-key-id")
-    p.add("--source-aws-secret-access-key")
-    p.add("--source-aws-endpoint")
+    p.add_argument("--source-type", choices=["s3", "oci"], default="s3")
+    p.add_argument("--source-aws-bucket")
+    p.add_argument("--source-aws-key")
+    p.add_argument("--source-aws-region")
+    p.add_argument("--source-aws-access-key-id")
+    p.add_argument("--source-aws-secret-access-key")
+    p.add_argument("--source-aws-endpoint")
```

Comment:
I ended up switching out the alias `add` for the actual method `add_argument`. IDE was able to pick up the typings better